src/module.indexer/model/dftx/compositeswap.ts

```diff
@@ -35,14 +36,22 @@ export class CompositeSwapIndexer extends DfTxIndexer<CompositeSwap> {
       poolIds.push({ id: poolPairToken.poolPairId })
     }
 
+    let swapAmount
```

Comment:
```suggestion
    let swapAmount: BigNumber
```
Since there is no type by reference here, should we explicitly add it here?